test/prism.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
The Travis builds are currently failing because this file does not have execute permissions.

Reply:
hmm, might be, I haven't seen chmod in any other projects travis.yaml files though.

Reply:
I'm not suggesting this be done via Travis. Try the following and commit the changes:
```chmod a+x test/prism.sh```
Note that this won't work if you're on a Windows box. If that's the case, then I can make the change.

Reply:
I think it doesn’t matter as executable flag is stored in inode and it’s not a part of the file.

Reply:
Correct. This particular issue is resolved with https://github.com/sendgrid/sendgrid-java/pull/229/commits/a21f3da020ef4ff8eaa01c4ccc9952cf7006b3a1

Reply:
@andy-trimble I still don't understand how this could solve the problem, permissions are stored in inode which is a structure that is stored somewhere on OS. chmod shouldn't change the file itself. Could you plz explain where my understanding is wrong?

Reply:
@dmitraver It is no longer a problem. You fixed it with commit https://github.com/sendgrid/sendgrid-java/commit/a21f3da020ef4ff8eaa01c4ccc9952cf7006b3a1.

Reply:
@andy-trimble yeah but for me this not enough, I want to figure out why its fixed behind the curtains. Just curious ;)

Reply:
@dmitraver At the time that I made this comment, the Travis build was failing with the following message `/home/travis/.travis/job_stages: line 57: ./test/prism.sh: Permission denied` This indicates that the execute permission on the file were not present. Github preserves file permissions, and when Travis checked it out to execute the build, it was not executable.

Reply:
@andy-trimble yup, I figured out that git actually preserves the file permissions .
git cat-file -p $hash permission bits are stored there as well.